vendor/milesight-iot/em500-udl-codec.yaml

```diff
@@ -0,0 +1,22 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: em500-udl.js
+  examples:
+    - description: Ultrasonic Distance Level Sensors (example 1) - Milesight IoT
+      input:
+        fPort: 85
+        bytes: [0x01, 0x75, 0x5C, 0x03, 0x82, 0x1e, 0x00]
+      output:
+        data:
+          battery: 92
+          distance: 30

```

Comment:
your decoder file is reads 0.03 instead of 30, please change the value here to 0.03 or review your .js file